src/Microsoft.Health.Fhir.Api/Resources.resx

```diff
@@ -189,11 +189,14 @@
   <data name="UrlResourceIdMismatch" xml:space="preserve">
     <value>Id in the URL must match id in the resource.</value>
   </data>
+  <data name="ValueCannotBeNull" xml:space="preserve">
+    <value>Value cannot be null.  Parameter name: {0}.</value>
```

Comment:
Can you add a resource comment on what this is supposed to be to help with translation?